### PR TITLE
Privacy Policy and Terms of Service

### PR DESCRIPTION
**Pull request by rinilkunhiraman**
Pages for the privacy policy and terms of
service added. Routes legal, legal/privacy-policy
and legal/terms-of-service created.Basic styles added.

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -48,15 +49,25 @@ const Layout = ({ children, boxed = true }) => {
...
                     buttonText="Agree"
                     cookieName="ownemployed-cookie-accepted"
+                    enableDeclineButton
+                    onDecline={() => {
+                        alert('nay!')
+                    }}
                     contentStyle={{ flex: 'none' }}
                     location="bottom"
                     style={{ justifyContent: 'center', background: '#124780' }}
+                    debug={true}
```
Comment (nopestack) [inline src/components/Layout.tsx]: Would you set this from an environment variable instead?

File: src/pages/privacy/privacyPolicy.tsx [hunk 2]
```diff
@@ -0,0 +1,704 @@
+import React from 'react'
```
Comment (nopestack) [inline src/pages/privacy/privacyPolicy.tsx:1]: The content on this page is rather long. Would you put it in a JSON outside of this component, import it and render it that way?

File: src/pages/privacy/termsOfService.tsx [hunk 3]
```diff
@@ -0,0 +1,571 @@
+import React from 'react'
```
Comment (nopestack) [inline src/pages/privacy/termsOfService.tsx]: Same suggestion here.

**Review by nopestack — CHANGES_REQUESTED** [review]

File: src/components/Layout.tsx (hunk 1 above)
Comment (rinilkunhiraman) [reply src/components/Layout.tsx]: ok. I will do that

**Comment by rinilkunhiraman** [comment]
I just noticed that there is already a commit with the terms of use with a merge conflict. Kindly have a look https://github.com/Ownemployed-Team/ownemployed/tree/master/src/pages/termsOfUse

**Comment by nopestack** [comment]
@karvex would you look at this and see how we can address these conflicts?

File: src/pages/privacy/privacyPolicy.tsx (hunk 2 above)
Comment (segirim) [reply src/pages/privacy/privacyPolicy.tsx:1]: Why not use markdown all the way for crazy long text? There is markdown editors we can use. I think we will also use services like contentful etc in the future and they support markdown. Look at termsOfUse.tsx. I think it's gonna be a very very time consuming thing every time we have long text.

File: src/pages/privacy/termsOfService.tsx (hunk 3 above)
Comment (segirim) [reply src/pages/privacy/termsOfService.tsx]: Same as above

File: src/routes.ts [hunk 4]
```diff
@@ -40,8 +40,8 @@ export default [
         component: CreateProjectPage,
         path: '/create-project',
     },
-    {
-        component: TermsOfUse,
-        path: '/terms-of-use',
-    },
+    // {
```
Comment (nopestack) [inline src/routes.ts]: Remove this commented code

**Review by nopestack — APPROVED** [review]

**Pull request merged**